src/opnsense/mvc/app/models/OPNsense/Proxy/Proxy.xml

```diff
@@ -182,6 +182,24 @@
                     </Constraints>
                 </perHostTrotteling>
             </traffic>
+            <parentproxy>
+                <enabled type="BooleanField">
+                    <default>0</default>
+                    <Required>Y</Required>
+                </enabled>
+                <host type="HostnameField">
+                    <Required>N</Required>
+                </host>
+                <port type="PortField">
+                    <Required>N</Required>
+                </port>
```

Comment:
you may want to use my SetIfContraint

Reply:
Thanks, nice idea, I did it with DependConstraint